README.md

```diff
@@ -79,9 +79,26 @@ Some secrets and sensitive configuration files are needed to properly operate ce
    ...
    ```
 
+1. Add a `cypress.env.json` in the `apps/mull-ui-e2e/` project folder. See `docs-and-resources` channel on slack for our secret key.
+
+   ```properties
+   {
+      "ACCESS_TOKEN_SECRET": "<secret-key>"
+   }
+   ```
+
+1. Add a `host.env.ts` under the `libs/types/src/` folder. This file specify the hostnames where the application are currently being hosted, and makes it easy to change them.
+
+   ```ts
+   export const UI_HOSTNAME = 'hostname';
+   export const API_HOSTNAME = 'hostname';
+   ```
```

Comment:
is there a particular reason why these are not added to .env file? I'm not sure there's a need to add another file.

Reply:
I also noticed that all but one uses http://. I think we can set it by default to be http://localhost: in the .env file and developers can change the localhost to what ever they want without the need to add another file.

Reply:
Oh dang, I must have just skipped that part of the description sorry. 

can I ask why do we need to ignore the ts file if we just set the two string to 'http://localhost' ?

Reply:
As discussed on slack, I'll be going with your suggestion @osahtout. I'll be removed the file from gitignore so that other don't need to set it up. We'll just need to keep in kind to not commit changes to that file.

Reply:
However, I think its fine to keep the protocol out of the name (http://), since its not what I want to make configurable, just  the host name